reports/3pc-consistency/3pc-consistency.tex

```diff
@@ -0,0 +1,92 @@
+\documentclass[acmlarge, screen, nonacm]{acmart}
+\usepackage[utf8]{inputenc}
+\usepackage{float}
+\usepackage{setspace}
+\usepackage{pgfplots}
+\usepackage{graphicx}
+\usepackage{fancyvrb}
+\usepackage{listings}
+\usepackage{xcolor}
+\usepackage{hyperref}
+  \hypersetup{colorlinks=true,allcolors=blue!40!black}
+\setlength{\topskip}{6pt}
+\setlength{\parindent}{0pt} % indent first line
+\setlength{\parskip}{6pt} % before par
+
+\title{Three phase commit consistency evaluation for git repository replication}
+
+\author{Ilya Moskvitin}
+\email{TODO: put your email address here}
+
+\author{Kirill Chernyavskiy}
+\email{g4s8.public@gmail.com}
+
+\acmBooktitle{none}
+\acmConference{none}
+\editor{none}
+
+\begin{document}
+
+\begin{abstract}
+  For git repository replication distributed system we need to use atomic
+  commit implementation based on git reference transaction hooks (todo: add reference to previous report).
+  There are three most popular implementations: two-phase commit (2PC), three-phase commit (3PC),
+  and Paxos commit protocols. 2PC protocol is not really fault tolerant, since it may turn storage
```

Comment:
Here could be proof-link to related work/paper

Reply:
You're right, I added a `todo` at 34 line, it was described the problem of 2PC in section 3.3 of "J. Gray and L. Lamport, Consensus on transaction commit" paper.